crates/common/flockfile/src/unix.rs

```diff
@@ -89,6 +104,15 @@ impl Flockfile {
         fs::remove_file(&self.path)?;
         Ok(())
     }
+
+    /// Create the lock file if it does not exist,
+    /// making sure any user can use this file as a lock.
+    fn create_world_writable_lock_file(path: &PathBuf) -> Result<(), io::Error> {
+        let file = File::create(path)?;
+        let permissions = fs::Permissions::from_mode(0o666);
+
```

Comment:
Remove the empty line.

Reply:
I love empty lines ;-) Seriously, I used them to separate logic blocks.